app/delegates/ContributionsDelegate.java

```diff
@@ -179,6 +179,8 @@ public static Integer countBy(String byUuid, Long byThemeId) {
                             sorting +=", creation asc nulls last";
                         } else if (sortingValue.equals("date_desc")) {
                             sorting +=", creation desc nulls last";
+                        } else if (sortingValue.equals("most_commented")) {
```

Comment:
@rebearteta 
To make it more general, let's also have a: 
- [ ]  "most_commented_public" => sorted by forumCommentCount
- [ ] "most_commented_members" => sorted by commentCount

This way the UI has the possibility to include or not the comments from the public or viceceversa. We can implement an app using only one of the resource spaces.